--- /Users/fiedl/python.cmake	2019-03-11 15:10:28.000000000 +0100
+++ python.cmake	2019-03-11 15:16:59.000000000 +0100
@@ -83,7 +83,7 @@
   PATHS
     ${PYTHON_ROOT}/include
   PATH_SUFFIXES
-    python${PYTHON_STRIPPED_MAJOR_MINOR_VERSION}
+    python${PYTHON_STRIPPED_MAJOR_MINOR_VERSION} python${PYTHON_STRIPPED_MAJOR_MINOR_VERSION}m
   NO_DEFAULT_PATH
 )
 # required for ubuntu, because their version of FindPythonLibs is different
